db/migrate/20200325170358_add_domain_and_relative_path_to_automate.rb

```diff
@@ -0,0 +1,124 @@
+require 'ancestry'
+
+class AddDomainAndRelativePathToAutomate < ActiveRecord::Migration[5.1]
+  class MiqAeDomain < ActiveRecord::Base
+    self.table_name = 'miq_ae_namespaces'
+  end
+
+  class MiqAeNamespace < ActiveRecord::Base
+    has_many :ae_classes, :class_name => "MiqAeClass", :foreign_key => :namespace_id
```

Comment:
The class name here needs to be fully qualified (i.e. `"AddDomainAndRelativePathToAutomate::MiqAeClass"`  (same with the other class references below).